builtin/logical/pki/path_revoke.go

```diff
@@ -414,6 +433,39 @@ func (b *backend) pathRotateCRLRead(ctx context.Context, req *logical.Request, _
 	}, nil
 }
 
+func (b *backend) pathRotateDeltaCRLRead(ctx context.Context, req *logical.Request, _ *framework.FieldData) (*logical.Response, error) {
+	sc := b.makeStorageContext(ctx, req.Storage)
```

Comment:
I'm curious what the thoughts are here about locks. The builder itself is locked, but `pathRotateCRLRead` grabs the storage lock, which I think feels wrong.

Reply:
I'm in agreement, if anything should be grabbing a lock it should be within the crlBuilder's code. Probably something I missed while refactoring?